src/solvers/dgmulti/sbp.jl

```diff
@@ -455,21 +497,28 @@ function calc_volume_integral!(du, u, mesh::DGMultiMesh,
           u_j = u[j]
           A_ij = vals[id]
           AF_ij = 2 * A_ij * volume_flux(u_i, u_j, normal_direction, equations)
-          du_i = du_i + AF_ij * inv_wq[i]
+          du_i = du_i + AF_ij
```

Comment:
Why don't we need this anymore? There are `AbstractPeriodicDerivativeOperator`s with non-uniform weights, e.g., periodic CGSEM operators.

Reply:
See https://github.com/trixi-framework/Trixi.jl/pull/1117#discussion_r846269539. Essentially we pre-multiply the inverse weights into the operator. 

This would have to be modified if we were to implement non-conservative fluxes; I'll add a comment on this.

Reply:
COmment added in https://github.com/trixi-framework/Trixi.jl/pull/1117/commits/7bc2e1e1527aaa94e61814df4fa64c655597cf96